benchmarks/benchmarks.py

```diff
@@ -0,0 +1,37 @@
+# Write the benchmarking functions here.
+# See "Writing benchmarks" in the asv docs for more information.
+import asyncio
+from .runner import *
+
+class TimeSuite:
+    """
+    An example benchmark that times the performance of various kinds
+    of iterating over dictionaries in Python.
+    """
+    def setup(self, nworkers =1):
+        ports = random_ports(nworkers + 1)
+        proxy_port = ports.pop()
```

Comment:
Let's try here:
```suggestion
        proxy_port = ports.pop()
        run_sync = asyncio.get_event_loop().run_until_complete
```

Then we can call this below, e.g.

```python
proxy = run_sync(start_proxy(...))
```

```python
prefix = run_sync(f)
```


```python
self.routes = run_sync(proxy.get_all_routes())
```

Reply:
Thanks :) Updated the changes. Now the benchmarks are running. Kindly, check.